client/src/pages/Details.js

```diff
@@ -76,5 +76,14 @@ const Container = styled.div`
 
   
 `;
+//added this here but not really sure what its doing--- found in docs
+export async function getServerSideProps() {
+  // Fetch data from external API
+  const res = await fetch(`/api/food`);
+  const feed = await res.json();
+
+  // Pass data to the page via props
+  return { props: { feed } };
+}
```

Comment:
`getServerSideProps` is a `Next.js` function that facilities the assembly of props on the server prior to creating the HTML markup for the initial page.  You can read more about it [here](https://nextjs.org/docs/basic-features/data-fetching/get-server-side-props)

Ultimately, you're not using `Next.js` in this project so you don't need it here.

```suggestion
```